### ISSUE TITLE
Last saved settings are not being remembered! (v4.3.3 - macOS Sequioa)

### ISSUE THREAD

**Issue by anand2996** [MonitorControl/MonitorControl#1686]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Intended use:

- My mac is connected to an external display. Once connected, I want the mac laptop's display to go to the lowest brightness possible (i.e, making it completely black to save display life)

Problem:

- I have set individual birghtness levels for both the external display (decent brightness) and the laptop display (lowest level - fully black)
- Now, everytime the display goes to sleep and I wake it up either with mouse click or mouse movement, then BOTH the displays come awake.
- When this happens, the brightness level of the external display is remembered, however the mac laptop display also comes awake with some level of brightness and I have to bring the slider to 0 from the top menu to switch it off.

Misc. Notes:

- Upon startup and wake: Assume last saved settings are valid - is the option I have selected and what should technically be syncing and making this behaviour possible, however it doesn't seem to be remembering the laptop displays brightness setting.

Requesting help to solve this as I don't want to have to bring the brightness to 0 everytime I wake the laptop.

Other than this MonitorControl is absolutely a LOVELY app and has worked better for me personally than BetterDisplay!

### Steps to reproduce

- Set individual birghtness levels for both the external display (decent brightness) and the laptop display (lowest level - fully black)
- Now, everytime the display goes to sleep, wake it up either with mouse click or mouse movement, then BOTH the displays come awake.
- When this happens, the brightness level of the external display is remembered, however the mac laptop display also comes awake with some level of brightness and I have to bring the slider to 0 from the top menu to switch it off.

PS: In the general settings make sure "Upon startup and wake: Assume last saved settings are valid" is selected - this should technically be syncing and making this behaviour possible, however it doesn't seem to be remembering the laptop displays brightness setting.

### Expected behavior

My mac is connected to an external display. Once connected, I want the mac laptop's display to go to the lowest brightness possible (i.e, making it completely black to save display life)

- Once I have already set the laptop display to the lowest brightness, I want it to remember this setting permanently.
- When I wake the screen, it should not come on again

### Environment Information (please complete the following information)

```markdown
- macOS version: Sequioa 15.1
- Mac model: Macbook Pro (14-inch, 2023)
- MonitorControl version: 4.3.3
- Monitor(s): Dell S2722QC
- Apple Silicon/M1 (yes or no): Yes (M3 Pro)
```

**Comment by waydabber** [comment]
Yes, the app will take the system configured brightness granted for the built-in display. You can use BetterDisplay (my other app) instead of MonitorControl where you have the option to override the built-in display's brightness on wake based on the last set value.
